### ISSUE TITLE
Fails to uninstall on fedora 41

### ISSUE THREAD

**Issue by Department-stack** [GyulyVGC/sniffnet#644]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### What's the problem?

![screenshot-2024-11-25-21-33-07](https://github.com/user-attachments/assets/8b234df8-81f3-42e1-b675-6f8eaa0951fb)

### How did you install the app?

Downloaded one of the provided packages

### Operating System

Linux

### Additional context

Distribution: Fedora KDE 41

Desktop environment: COSMIC Alpha 3

Kernel version: 6.11.8-300.fc41.x86_64

Processors: 2 × AMD A4-9120C RADEON R4, 5 COMPUTE CORES 2C+3G

Memory: 3.5 GiB of RAM

Graphics: AMD Radeon R3 Graphics

Network adapter: Qualcomm Wireless-AC 802.11a/b/g/n/ac (2x2) Wi-Fi and Bluetooth 4.2 Combo, non-vPro

Laptop model: HP Chromebook 11A G8 EE

**Comment by GyulyVGC** [comment]
Hey, thanks for the feedback, this was just fixed in https://github.com/GyulyVGC/sniffnet/commit/a1f115ab1105df5b353171a01fecdb37b1642d23.

**Comment by deividt** [comment]
Hi, it seems like this fix didn’t work for me.
Am I missing something?

I am running v1.3.2 on Fedora.
And installed from the rpm package available here: https://sniffnet.net/download/

![Image](https://github.com/user-attachments/assets/19551246-c60e-4287-813a-5370bb3f4771)

**Comment by Department-stack** [comment]
can reproduce

Laptop model: Acer Chromebook Plus 514 (CB514-3HT-R8RB)
OS: Fedora Linux 41 (KDE Plasma) x86_64
Host: Markarth (rev3)
Kernel: Linux 6.12.9-200.fc41.x86_64
Shell: bash 5.2.32
Display (AUO9CAB): 1920x1200 @ 60 Hz in 14" [Built-in]
DE: COSMIC
CPU: AMD Ryzen 3 7320C (8) @ 2.40 GHz
GPU: AMD Mendocino [Integrated]
Network Adapter: MediaTek MT7922A22M
Memory: 7.59 GiB
Swap: 7.59 GiB

![Image](https://github.com/user-attachments/assets/5d168acf-2198-411c-ab08-06cd900fb1e3)

**Comment by GyulyVGC** [comment]
Mm that's weird guys.
Could you please try to run the command `setcap -r /usr/bin/sniffnet` manually?

**Comment by Department-stack** [comment]
![Image](https://github.com/user-attachments/assets/4bae5fdb-2299-41b3-a648-7ef62ff47314)

**Comment by GyulyVGC** [comment]
Ok so that command is not giving errors, which is the same command that I'm using in the pre-remove script. I'm wondering why it fails when run from the pre-remove instead.

**Comment by deividt** [comment]
Yeah, but to me it gives a different result when running with sudo.
I'm also sending the permissions of the file, not sure if it helps.

![Image](https://github.com/user-attachments/assets/e692aa82-1b31-4b5c-91da-f1f2f7463fa0)

![Image](https://github.com/user-attachments/assets/bdd6a9a7-a729-4b4c-94bb-5feb2fd34731)
![Image](https://github.com/user-attachments/assets/0feff6e2-f3ed-4132-854d-8a99a9d5194a)

**Comment by GyulyVGC** [comment]
At this point I'm just going to remove the pre-uninstall script.
We included it as part of the packaging material, but I wasn't even sure it was strictly needed.
Everything should work even without resetting capabilities before the uninstallation.
